modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -1,7 +1,189 @@
 [id='ref-rbac-rest-api-endpoints_{context}']
 = Supported RBAC REST API endpoints
 
-The RBAC REST API provides the following endpoints for managing permission policies and roles in the {product-short} and for retrieving information about the policies and roles.
+The RBAC REST API provides the following endpoints for managing roles, permission and, conditional policies in the {product-short} and for retrieving information about the roles and policies.
+
+== Roles
+
+The RBAC REST API supports the following endpoints for managing roles in the {product}.
+
+[GET] /api/permission/roles::
++
+--
+Returns all roles in {product-short}.
+
+.Example response (JSON)
+[source,json]
+----
+[
+  {
+    "memberReferences": ["user:default/pataknight"],
+    "name": "role:default/guests"
+  },
+  {
+    "memberReferences": [
+      "group:default/janus-authors",
+      "user:default/matt"
+    ],
+    "name": "role:default/rbac_admin"
+  }
+]
+----
+--
+
+[GET] /api/permission/roles/{kind}/{namespace}/{name}::
```

Comment:
```suggestion
[POST] /api/permission/roles/{kind}/{namespace}/{name}::
```
I think somewhere the GET endpoint for single roles and POST endpoint got merged. So this is basically the POST and we need to include the GET endpoint as well.

```
[GET] /api/permission/roles/{kind}/{namespace}/{name}::
+
--
Returns information for a single role  {product-short}.

.Example response (JSON)
[source,json]
----
[
  {
    "memberReferences": [
      "group:default/janus-authors",
      "user:default/matt"
    ],
    "name": "role:default/rbac_admin"
  }
]
----
--
```

Reply:
Done